politeiawww/user/user.go

```diff
@@ -478,4 +482,8 @@ type Database interface {
 
 	// Close performs cleanup of the backend.
 	Close() error
+
+	// SetQuiesce sets userdb quiesce mode toggle value, if Quiesced all writes
+	// are blocked, only reads all allowed.
+	SetQuiesce(quiesce bool) error
```

Comment:
Put this above `Close()`.  It makes sense to have `Close()` as the last method being defined.